docs/schema/2.0/manifest.schema.json

```diff
@@ -224,6 +224,9 @@
       },
       "required": ["scopes", "publicKeys"]
     },
+    "opdpDataProvider": {
+      "$ref": "./opdpDataProvider.schema.json"
+    },
```

Comment:
My only question here is whether I should move the entire opdp provider schema into a separate file, or just the opdp manifest. So here it could be like this: 
```
{
  "type": "object",
  "additionalProperties": false,
  "properties": {
    "manifest-url": {
      "type": "string",
      "format": "uri-template"
    },
    "manifest": {
      "$ref": "./opdpProviderManifest.schema.json"
    },
  },
  "required": ["manifest", "manifest-url"]
}
```

WDYT @PeSme?